aya/src/bpf.rs

```diff
@@ -189,6 +191,29 @@ impl<'a> BpfLoader<'a> {
         self
     }
 
+    /// Allows bytecode containing maps unsupported by Aya to be loaded.
+    ///
+    /// By default, programs containing maps unsupported by Aya will not be loaded.
+    /// This function changes the default behavior, allowing programs to be loaded.
+    /// This should only be used in cases where you do not require access to eBPF
```

Comment:
Allows programs containing unsupported maps to be loaded.

By default programs containing unsupported maps will fail to load. This method
can be used to configure the loader so that unsupported maps will be loaded, but
won't be accessible from userspace. Can be useful when using unsupported
maps that are only accessed from eBPF code and don't require any userspace
interaction.